### PR TITLE
polish(publisher): receive publish operations on channel

### PR DESCRIPTION
**Pull request by frrist**
prevents tasks from blocking on publish

### CODE REVIEW

**Review by placer14 — APPROVED** [review]
LGTM

File: services/processor/publisher.go [hunk 1]
```diff
@@ -9,20 +9,37 @@ import (
...
+func (p *Publisher) Start(ctx context.Context) {
+	p.log.Info("starting publisher")
+	go func() {
+		for {
+			select {
+			case <-ctx.Done():
+				p.log.Info("stopping publisher")
+				return
+			case persistable := <-p.pubCh:
+				go func() {
```
Comment (iand) [inline services/processor/publisher.go:35]: I have a slight worry about the unbounded concurrency of starting a goroutine for each persistence job, mainly due to the use of database connections from the pool. Something to look at as part of tuning later.

**Review by iand — APPROVED** [review]
LGTM just a couple of comments for future

File: services/processor/publisher.go (hunk 1 above)
Comment (frrist) [reply services/processor/publisher.go:35]: Agreed. I initially considered adding throttling based on a connection pool size, but doing so this early felt premature.

**Pull request merged**